server/src/controllers/Events/resolver.ts

```diff
@@ -710,17 +711,31 @@ ${venueDetails}`;
     });
     await deleteEventReminders(id);
 
+    const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter.id}`;
+    const eventURL = `${process.env.CLIENT_LOCATION}/events/${event.id}?emaillink=true`;
     const notCanceledRsvps = event.event_users;
 
     if (notCanceledRsvps.length) {
       const emailList = notCanceledRsvps.map(({ user }) => user.email);
-      const subject = `Event ${event.name} canceled`;
-      const body = `The event ${event.name} was canceled`;
+      const subject = `Event ${event.name} is canceled`;
+      const cancelEventEmail = `The Upcoming Event for ${event.name} is canceled.<br />
```

Comment:
```suggestion
      const cancelEventEmail = `The Upcoming Event ${event.name} is canceled.<br />
```
With _for_ it looks like ${event.name} is supposed to be the chapter instead.